content/guides/component-testing/mounting-react.md

```diff
@@ -14,105 +14,103 @@ end-to-end test.
 ## What is the Mount Function?
 
 We ship a `mount` function for each front-end framework Cypress supports, which
-can be imported from the `cypress` package. It is responsible for rendering
-components within Cypress's sandboxed iframe as well as handling and
-framework-specific cleanup.
+is imported from the `cypress` package. It is responsible for rendering
+components within Cypress's sandboxed iframe and handling and framework-specific
+cleanup.
 
 ```js
 import { mount } from 'cypress/react'
 ```
 
-### The Simplest Usage of Mount
+### Using cy.mount Anywhere
```

Comment:
We use parens when we refer to the mount command on the mount API page. I might consider these for consistency 

```suggestion
### Using `cy.mount()` Anywhere
```